src/App.jsx

```diff
@@ -14,6 +16,84 @@ export const goods = [
   'Garlic',
 ];
 
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+
+  function buttonSwitch(good) {
+    setSelectedGood(prev => (prev === good ? null : good));
+  }
+
+  function clearButton() {
+    setSelectedGood(null);
+  }
+
+  const isAnyGoodSelected = selectedGood !== null;
+
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood ? `${selectedGood} is selected` : 'No goods selected'}
+
+        {selectedGood && (
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={clearButton}
+          />
+        )}
+      </h1>
+      <table className="table">
+        <tbody>
+          {goods.map(good => {
+            let button;
+
+            if (isAnyGoodSelected && selectedGood === good) {
```

Comment:
do not use let, if, else in jsx.
just ternary and logical operators for conditinal rendering